tests/eva_cttv_pipeline/evidence_string_generation/test_clinvar_to_evidence_strings.py

```diff
@@ -180,3 +182,62 @@ def test_somatic_evidence_string(self):
         evidence_string = json.dumps(evidence, sort_keys=True, indent=2)
         expected_evidence_string = open(config.expected_somatic_evidence_string).read()
         self.assertEqual(evidence_string, expected_evidence_string)
+
+
+class GroupDiseasesByMappingTest(unittest.TestCase):
```

Comment:
This might be a bit of a tall order to ask, but do you think we could get rid of unittest completely as part of this PR and just use pytest everywhere?

If this is too much work, then it can totally be extracted into a separate follow-up issue, no problem

Reply:
I'm 100% behind switching over to pytest completely (and part of me want to just do it now), but I think it would be wiser to get this PR merged and make a follow-up issue as it's not a trivial amount of work.

Reply:
Agreed! Created a follow-up issue here: https://github.com/EBIvariation/eva-opentargets/issues/231